blog/release-v1.16/index.mdx

```diff
@@ -0,0 +1,238 @@
+metadata:
+  name: imageprepull-example
+  labels:
+    description:ImagePrePullLabel
+spec:
+  imagePrePullTemplate：
+    images:
+      - image1
+      - image2
+    nodes：
+      - edgenode1
+      - edgenode2
+    checkItems:
+      - "disk"
+    failureTolerate: "0.3"
+    concurrency: 2
+    timeoutSeconds: 180
+    retryTimes: 1
+```
+
+Refer to the link for more details. ([#5310](https://github.com/kubeedge/kubeedge/pull/5310), [#5331](https://github.com/kubeedge/kubeedge/pull/5331))
+
+### Support Installing Windows-based Edge Nodes with Keadm
+
+KubeEdge has supported the edge node running on Windows Server 2019 in v1.15, extending KubeEdge to the Windows ecosystem and expanding its use cases and ecosystem.
+
+In this release, Windows-based Edge Nodes can be installed and registered to cloud with the installation tool `Keadm`, providing convenience for the application of KubeEdge in Windows OS.
+
+Refer to the link for more details. ([#4968](https://github.com/kubeedge/kubeedge/pull/4968))
+
+### Add Compatibility Tests for Multiple Runtimes
+
+The e2e test of KubeEdge v1.16 has integrated compatibility tests for multiple container runtimes. Currently, four container runtime compatibility tests have been added, including **containerd**, **docker**, **cri-o**, and **isulad**.
+
+Refer to the link for more details.([#5321](https://github.com/kubeedge/kubeedge/pull/5321))
+
+### Support More Deployment Fields to the EdgeApplication Overrides
+
+In previous versions, only replicas and image of the EdgeApplication could be overridden. In this release, we support overriding more Deployment fields: env, command, args and resources.
+
+Refer to the link for more details.([#5038](https://github.com/kubeedge/kubeedge/pull/5038))
+
+### Support Mapper Upgrade
+
+Build mapper upgrade framework. Users can upgrade the mapper by changing the referenced mapper-framework package version.
+
+- **Mapper-framework code decouple**
+
+    The code in mapper-framework was decoupled into user-layer code and business-layer code, and create the [kubeedge/mapper-framework](https://github.com/kubeedge/mapper-framework) repo to store the business layer code.
+
+- **Mapper upgrade framework**
+
+    Update the way mapper-framework generates mapper projects. The current execution script will only generate user-level code through dependent references. When the mapper project needs to be upgraded, it can be directly made by changing the version of mapper-framework package.
+
+Refer to the link for more details.([#5308](https://github.com/kubeedge/kubeedge/pull/5308), [#5326](https://github.com/kubeedge/kubeedge/pull/5326))
+
+### Integrate Redis and TDengine Database in DMI Data Plane
+
+Integrate redis and tdengine database in DMI data plane. The mapper project generated by mapper-framework has build-in ability to push data to redis and tdengine database. Users can push data directly through configuring device instance files.
+
+Database Field Definition:
+```json
```

Comment:
The language can be specified as golang.  ```golang

Reply:
I compared json with golang. I think `json` is more suitable both in display and its meaning here.

![image](https://github.com/kubeedge/website/assets/48788150/74c1e755-8f21-48b2-b02f-c0c3468e50f7)

Reply:
Okey